### PR TITLE
Add vs code extension

### PR DESCRIPTION
**Pull request by podhrmic**
Very simple syntax highligher for Lando

### CODE REVIEW

File: lando.vs-code.0.1/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# lando README
+
+This is the README for your extension "lando". After writing up a brief description, we recommend including the following sections.
```
Comment (kiniry) [inline lando.vs-code.0.1/README.md]: How about "Lando"?

File: lando.vs-code.0.1/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "lando syntax highlighting",
```
Comment (kiniry) [inline lando.vs-code.0.1/package.json]: "Lando" etc.

**Review by kiniry — APPROVED** [review]
I suggest making those tweaks for capitalization then you can merge.

File: lando.vs-code.0.1/README.md (hunk 1 above)
Comment (podhrmic) [reply lando.vs-code.0.1/README.md]: Done

File: lando.vs-code.0.1/package.json (hunk 2 above)
Comment (podhrmic) [reply lando.vs-code.0.1/package.json]: Done

**Pull request merged**